### PR TITLE
UIQM-205: Improve handling of parsing multiple subfields

### PR DESCRIPTION
**Pull request by mariiaschv**
## Purpose
Fix MARC row is not saved when every subfield code in this row is not separated by space with its value (e.g. `$atesting $btestingtwo`).

## Approach
It was happening due to regex in `checkIsEmptyContent` function which considered field content empty if all of its space separated parts started with `$` followed by zero to **unlimited** number of characters. This considered both `$atesting` and `$btestingtwo` parts as not holding any values.
What needed to be done is to alter regex to consider field content empty only if all of its space separated parts started with `$` followed by zero to **one** number of characters. Thus considering `$atesting` and `$btestingtwo` parts to have values but `$a` and `$b` to have empty values.

## Issues
[UIQM-205](https://issues.folio.org/browse/UIQM-205)

### CODE REVIEW

File: src/QuickMarcEditor/utils.js [hunk 1]
```diff
@@ -452,7 +452,7 @@ export const removeFieldsForDuplicate = (formValues) => {
 
 const checkIsEmptyContent = (field) => {
   if (typeof field.content === 'string') {
-    return compact(field.content.split(' ')).every(content => /^\$[a-z0-9]*/.test(content));
+    return compact(field.content.split(' ')).every(content => /^\$[a-z0-9]?$/.test(content));
```
Comment (BogdanDenis) [inline src/QuickMarcEditor/utils.js:455]: Please add a couple of tests for this - check that fields with empty content are removed and content like `$atesting $btestingtwo` is not considered to be empty

**Review by BogdanDenis — APPROVED** [review]

**Review by Svitlana-Korol — APPROVED** [review]
